charts/crd/crd-operating-system-config.yaml

```diff
@@ -76,6 +76,9 @@ spec:
                       - content
                       - path
                     properties:
+                      append:
```

Comment:
Why this is needed? Is it used somewhere in your PR ?

Reply:
Yes, it's used in here:
https://github.com/kubermatic/operating-system-manager/blob/b8d40e0cf74ad4d352f6a4810d30a9a294fcf9e6/examples/osp-sles.yaml#L467

Reply:
Yeah but again those configs are not needed, please remove any configs which are added by this PR https://github.com/kubermatic/machine-controller/pull/981

Reply:
Makes sense, i'll remove this.